src/components/common/AddEditOrgPerson.vue

```diff
@@ -82,7 +82,7 @@
                   label="Middle Name (Optional)"
                   id="person__middle-name"
                   v-model.trim="orgPerson.officer.middleName"
-                  :rules="enableRules ? Rules.MiddleNameRules: []"
+                  :rules="enableRules ? Rules.MiddleNameRules20Char: []"
```

Comment:
I think it would actually be better to make firms the special case, since FirstNameRules is used for other things (like Signing Party and Custodian of Records).

Otherwise the implementation looks OK to me.

Reply:
This makes sense as only firms are kind of a special case. Updated accordingly.